tests/integration/cli/classification/test_classification.py

```diff
@@ -296,6 +296,18 @@ def test_otx_train(self, template, tmp_dir_path):
         tmp_dir_path = tmp_dir_path / "multi_label_cls"
         otx_train_testing(template, tmp_dir_path, otx_dir, args_m)
 
+    @e2e_pytest_component
+    @pytest.mark.parametrize("template", templates, ids=templates_ids)
```

Comment:
How about to run this test only on default template to reduce test time?

Reply:
In the state_dict_mixing algorithm, the logic could be different to model architectures. So I think checking all architectures could be safe. https://github.com/sungmanc/training_extensions/blob/312d51ed6de8182a9f3449ea65721bacc353ca9b/otx/algorithms/classification/adapters/mmcls/models/classifiers/sam_classifier.py#L244